### PR TITLE
Nightwatch project added

### PR DESCRIPTION
**Pull request by SridattaRishi**

### CODE REVIEW

File: selenium/javaScript/Chapter1:-nightwatch for sinlge browser/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+//exclude below files/dir
+
+node_modules/*
+reports/*
+lambda_api.log
```
Comment (shibupanda) [inline selenium/javaScript/Chapter1:-nightwatch for sinlge browser/.gitignore:5]: what is this lambda_api.log file? why it is lambda?
Is this log file really created

File: selenium/javaScript/Chapter2:-nightwatch for Multiple browser/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "JS-Nightwatch.js",
+  "version": "0.0.0",
+  "description": "NightWatch examples for pcloudy",
+  "main": "",
+  "scripts": {
+    "test": "./node_modules/.bin/nightwatch -e chrome,firefox,edge,safari tests"
```
Comment (shibupanda) [inline selenium/javaScript/Chapter2:-nightwatch for Multiple browser/package.json:7]: create different script for linux and win and use only that env which are in nightwatch.json

**Review by shibupanda — CHANGES_REQUESTED** [review]
Please don't upload package-lock.json

**Comment by SridattaRishi** [comment]
No this log is not created at all. I had tried but not got this log.

**Review by shibupanda — APPROVED** [review]

**Pull request merged**